### ISSUE TITLE
TypeError: Argument 1 passed to CompletionProvider::expandParentFqns() must be of the type array, null given.

### ISSUE THREAD

**Issue by jens1o** [felixfbecker/php-language-server#351]
```
[Error - 9:35:51 AM] Request textDocument/completion failed.
  Message: TypeError: Argument 1 passed to LanguageServer\CompletionProvider::expandParentFqns() must be of the type array, null given, called in C:\Users\Jens\.vscode\extensions\felixfbecker.php-intellisense-1.1.5\vendor\felixfbecker\language-server\src\CompletionProvider.php on line 325 and defined in C:\Users\Jens\.vscode\extensions\felixfbecker.php-intellisense-1.1.5\vendor\felixfbecker\language-server\src\CompletionProvider.php:319
Stack trace:
#0 C:\Users\Jens\.vscode\extensions\felixfbecker.php-intellisense-1.1.5\vendor\felixfbecker\language-server\src\CompletionProvider.php(325): LanguageServer\CompletionProvider->expandParentFqns(NULL)
#1 C:\Users\Jens\.vscode\extensions\felixfbecker.php-intellisense-1.1.5\vendor\felixfbecker\language-server\src\CompletionProvider.php(151): LanguageServer\CompletionProvider->expandParentFqns(Array)
#2 C:\Users\Jens\.vscode\extensions\felixfbecker.php-intellisense-1.1.5\vendor\felixfbecker\language-server\src\Server\TextDocument.php(362): LanguageServer\CompletionProvider->provideCompletion(Object(LanguageServer\PhpDocument), Object(LanguageServer\Protocol\Position))
#3 [internal function]: LanguageServer\Server\TextDocument->LanguageServer\Server\{closure}()
#4 C:\Users\Jens\.vscode\extensions\felixfbecker.php-intellisense-1.1.5\vendor\sabre\event\lib\coroutine.php(70): Generator->send(Object(LanguageServer\PhpDocument))
#5 C:\Users\Jens\.vscode\extensions\felixfbecker.php-intellisense-1.1.5\vendor\sabre\event\lib\Promise.php(242): Sabre\Event\{closure}(Object(LanguageServer\PhpDocument))
#6 C:\Users\Jens\.vscode\extensions\felixfbecker.php-intellisense-1.1.5\vendor\sabre\event\lib\Loop\Loop.php(261): Sabre\Event\Promise->Sabre\Event\{closure}()
#7 C:\Users\Jens\.vscode\extensions\felixfbecker.php-intellisense-1.1.5\vendor\sabre\event\lib\Loop\Loop.php(215): Sabre\Event\Loop\Loop->runNextTicks()
#8 C:\Users\Jens\.vscode\extensions\felixfbecker.php-intellisense-1.1.5\vendor\sabre\event\lib\Loop\Loop.php(194): Sabre\Event\Loop\Loop->tick(true)
#9 C:\Users\Jens\.vscode\extensions\felixfbecker.php-intellisense-1.1.5\vendor\sabre\event\lib\Loop\functions.php(122): Sabre\Event\Loop\Loop->run()
#10 C:\Users\Jens\.vscode\extensions\felixfbecker.php-intellisense-1.1.5\vendor\felixfbecker\language-server\bin\php-language-server.php(50): Sabre\Event\Loop\run()
#11 {main}
```

**Comment by lloy0076** [comment]
This happened to me as well:

```
[Error - 10:10:02 am] Request textDocument/completion failed.
  Message: TypeError: Argument 1 passed to LanguageServer\CompletionProvider::expandParentFqns() must be of the type array, null given, called in C:\Users\david_000\.vscode\extensions\felixfbecker.php-intellisense-1.3.0\vendor\felixfbecker\language-server\src\CompletionProvider.php on line 305 and defined in C:\Users\david_000\.vscode\extensions\felixfbecker.php-intellisense-1.3.0\vendor\felixfbecker\language-server\src\CompletionProvider.php:299
Stack trace:
#0 C:\Users\david_000\.vscode\extensions\felixfbecker.php-intellisense-1.3.0\vendor\felixfbecker\language-server\src\CompletionProvider.php(305): LanguageServer\CompletionProvider->expandParentFqns(NULL)
#1 C:\Users\david_000\.vscode\extensions\felixfbecker.php-intellisense-1.3.0\vendor\felixfbecker\language-server\src\CompletionProvider.php(151): LanguageServer\CompletionProvider->expandParentFqns(Array)
#2 C:\Users\david_000\.vscode\extensions\felixfbecker.php-intellisense-1.3.0\vendor\felixfbecker\language-server\src\Server\TextDocument.php(362): LanguageServer\CompletionProvider->provideCompletion(Object(LanguageServer\PhpDocument), Object(LanguageServer\Protocol\Position))
#3 [internal function]: LanguageServer\Server\TextDocument->LanguageServer\Server\{closure}()
#4 C:\Users\david_000\.vscode\extensions\felixfbecker.php-intellisense-1.3.0\vendor\sabre\event\lib\coroutine.php(70): Generator->send(Object(LanguageServer\PhpDocument))
#5 C:\Users\david_000\.vscode\extensions\felixfbecker.php-intellisense-1.3.0\vendor\sabre\event\lib\Promise.php(242): Sabre\Event\{closure}(Object(LanguageServer\PhpDocument))
#6 C:\Users\david_000\.vscode\extensions\felixfbecker.php-intellisense-1.3.0\vendor\sabre\event\lib\Loop\Loop.php(261): Sabre\Event\Promise->Sabre\Event\{closure}()
#7 C:\Users\david_000\.vscode\extensions\felixfbecker.php-intellisense-1.3.0\vendor\sabre\event\lib\Loop\Loop.php(215): Sabre\Event\Loop\Loop->runNextTicks()
#8 C:\Users\david_000\.vscode\extensions\felixfbecker.php-intellisense-1.3.0\vendor\sabre\event\lib\Loop\Loop.php(194): Sabre\Event\Loop\Loop->tick(true)
#9 C:\Users\david_000\.vscode\extensions\felixfbecker.php-intellisense-1.3.0\vendor\sabre\event\lib\Loop\functions.php(122): Sabre\Event\Loop\Loop->run()
#10 C:\Users\david_000\.vscode\extensions\felixfbecker.php-intellisense-1.3.0\vendor\felixfbecker\language-server\bin\php-language-server.php(50): Sabre\Event\Loop\run()
#11 {main}
  Code: -32603 
```

My culprit seems to be that I was trying to do:

```
use Laravel\Passport as Passport;

// autocomplete Passport::
```

But it's actually `Laravel\Passport\Passport`. Clearly this is an error in my code but a more "expected" solution would be something along the lines of "I couldn't complete that!" or "Passport:: is not completable!" rather than a rather jarring language server error.

I'm using "May 2017 (version 1.13) of VS Studio Code".
I'm using "v1.3.0" of PHP Intellisense but I don't know quite how to get the version of the language server - however it is recent as of 2017-06-13 00:00:00 -04:00.


![image](https://user-images.githubusercontent.com/1174532/27086704-07129b46-5021-11e7-8963-0c25f1521a16.png)

**Comment by felixfbecker** [comment]
Does this still happen in the latest version?

**Comment by lloy0076** [comment]
@felixfbecker - how does one tell if one has the latest version? AFAICT my VS Code is up to date (none of the modules in the extension view claim they need updating, the main interface doesn't claim it needs updating and Windows itself is up to date).

**Comment by felixfbecker** [comment]
The version is shown behind the name

![image](https://user-images.githubusercontent.com/10532611/27252081-9792adaa-5355-11e7-99d0-b05d51cfe602.png)

If it still happens, could you give reproduction steps?

**Comment by jens1o** [comment]
No, I can't reproduce it anymore.

**Comment by lloy0076** [comment]
1.4.1 seems to have fixed it; IIRC I was on a 1.3.? version before hand.
